code/mob/living/carbon/human/procs/get_desc.dm

```diff
@@ -346,7 +346,21 @@
 				. += "<br><span class='alert'>[src.name] has a blank expression on [his_or_her(src)] face.</span>"
 
 			if (!src.client && !src.ai_active)
-				. += "<br>[src.name] seems to be staring blankly into space."
+				var/using_vr_goggles = FALSE
+				var/mob = find_player(src.last_ckey)?.client?.mob
+
+				if (istype(mob, /mob/living/carbon/human/virtual))
+					using_vr_goggles = TRUE
```

Comment:
```suggestion
					var/mob/living/carbon/human/virtual/vr_person = mob
					if (!vr_person.isghost)
						using_vr_goggles = TRUE
```

Reply:
I guess which case would this fall under? You can only be an observer if you've died, in which case the dead message would appear first, or are an admin, release your body, and then go into vr space which I don't think that is going to occur very often

Reply:
Right, but this will eliminate those corner cases so why not add it?

Reply:
Sure